src/test/java/com/knu/buga1chuk/serialzation/service/JsonPersonSerializationServiceTest.java

```diff
@@ -0,0 +1,56 @@
+package com.knu.buga1chuk.serialzation.service;
+
+import com.knu.buga1chuk.model.Person;
+import com.knu.buga1chuk.model.PersonList;
+import com.knu.buga1chuk.serialization.service.JsonPersonSerializationService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class JsonPersonSerializationServiceTest {
+
+    @Test
+    void toPersonTest() {
```

Comment:
Wrong test name.

`should<mathod_name>On<extra_notes>`

Reply:
fix